easybuild/easyconfigs/f/fqtrim/fqtrim-0.9.4-intel-2016b.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+
+name = 'fqtrim'
+version = '0.9.4'
+
+homepage = 'http://ccb.jhu.edu/software/fqtrim/'
+description = """fqtrim is a versatile stand-alone utility that can be used to trim adapters, poly-A tails,
+ terminal unknown bases (Ns) and low quality 3' regions in reads from high-throughput next-generation sequencing
+ machines."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['http://ccb.jhu.edu/software/fqtrim/dl/']
+sources = [SOURCE_TAR_GZ]
+
+buildopts = 'release CC="$CXX" LINKER="$CXX" LDFLAGS="$LDFLAGS $CXXFLAGS"'
```

Comment:
I ran into a problem where the linking failed because `g++` was being used to link objects created by `icpc` (i.e., the `LINKER="$CXX"` was missing).

My first attempt to fix that was including `$CXXFLAGS` in `LDFLAGS`, but maybe that's indeed not required anymore now, I'll check, thanks.